ArduSub/AP_Arming_Sub.cpp

```diff
@@ -53,6 +53,37 @@ bool AP_Arming_Sub::arm(AP_Arming::Method method, bool do_arming_checks)
         return false;
     }
 
+    // if there's no shift assigned, it must not be a shift funtion
+    bool has_shift_btn = false;
```

Comment:
Scope can be tightened.

Reply:
it gave me an error when i tried adding an access modifier to the variable, so i'm not sure what else to do to tighten the scope

Reply:
I think your restructure made this request out-of-date.